### DISCUSSION TITLE
dpkg: error: control file 'md5sums' for package 'rustdesk' is missing value seperator

### DISCUSSION THREAD

**Discussion by gabefair** [rustdesk/rustdesk#10354] [General]
### Bug Description

rustdesk has incorrectly formatted md5 file in rustdesk package resulting in the system's inability to verify installation was successful.

### How to Reproduce

1) Download installer for Ubuntu
2) Double click on it to install it
3) Run `sudo dpkg --verify` in the terminal

### Expected Behavior

`sudo dpkg --verify` runs without issue

### Operating system(s) on local (controlling) side and remote (controlled) side

Ubuntu 24 -> Ubuntu 24

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.6 -> 1.3.6

### Screenshots

![image](https://github.com/user-attachments/assets/9517758a-d53b-435b-93eb-de82b3926900)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/10356
